genesis/example/member_domain_test.go

```diff
@@ -99,55 +98,42 @@ func (p *mockParentWithError) AddChild(child object.Child) (string, error) {
 	return "", fmt.Errorf("add child error")
 }
 
-type mockProxy struct {
-	mockCallable
-	parent object.Parent
-}
-
-func (p *mockProxy) GetClassID() string {
-	return "mockProxy"
-}
-
-func (p *mockProxy) GetParent() object.Parent {
-	return p.parent
-}
-
-type mockFactory struct {
-	mockCallable
+type mockFactoryProxy struct {
```

Comment:
Where is no need to name this variable `mockFactoryProxy`, because Factory is always a Proxy. `mockFactory` is more clear in this case.

Reply:
Sure